### ISSUE TITLE
LinearModelAnalysis::drawQQPlot line is not the first bisector

### ISSUE THREAD

**Issue by josephmure** [openturns/openturns#1573]
I am not sure this is a bug, because it seems quite deliberate in the code, but I find the line drawn by [LinearModelAnalysis::drawQQPlot](https://github.com/openturns/openturns/blob/12c832da852833d12a9a9a428641f5b82b624735/lib/src/Uncertainty/Algorithm/MetaModel/LinearModel/LinearModelAnalysis.cxx#L515) strange.

The line represented is not the first bisector, as I would expect. Here is a small bit of code to illustrate:
```
import openturns as ot
from openturns.viewer import View
ot.RandomGenerator.SetSeed(0)
X = ot.Uniform().getSample(30)
model = ot.SymbolicFunction(['X'], ['3*X + 2'])
Y = model(X) + ot.Normal().getSample(30)
lin = ot.LinearModelAlgorithm(X, Y)
lin.run()
analysis = ot.LinearModelAnalysis(lin.getResult())
View(analysis.drawQQplot())
```
The line drawn in the output graph does not go through the points (-2,-2) and(+2,+2) :
![not_bisector](https://user-images.githubusercontent.com/31989332/90027123-77c95d80-dcb8-11ea-93c6-c6109258e4f2.png)

Here is the corresponding source code:

```
  // add line to a normal QQ plot which passes through the first and third quartiles
  Sample diagonal(2, 2);
  Point point(2);
  const UnsignedInteger id1Q = 0.25 * size - 0.5;
  const UnsignedInteger id3Q = 0.75 * size - 0.5;
  Scalar delta = dataFull(id3Q, 0) - dataFull(id1Q, 0);
  // If quantiles are too close, bisectrice takes into account the min/max values
  if (!(delta > 0) && !(delta < 0))
    delta = dataFull(size - 1, 0) - dataFull(0, 0);
  point[0] = (dataFull(id3Q, 1) - dataFull(id1Q, 1)) / delta;
  point[1] = dataFull(id3Q, 1) - point[0] * dataFull(id3Q, 0);
  diagonal(0, 0) = dataFull(0, 0);
  diagonal(0, 1) = dataFull(0, 0) * point[0] + point[1];
  diagonal(1, 0) = dataFull(size - 1, 0);
  diagonal(1, 1) = dataFull(size - 1, 0) * point[0] + point[1];
  Curve curve(diagonal, "red", "dashed", 2);
```

Contrast with [VisualTest::drawQQPlot](https://github.com/openturns/openturns/blob/6cde44895358ade4033a74de0453f8131fe030ac/lib/src/Uncertainty/StatTests/VisualTest.cxx#L92), which does it differently:

```
  // First, the bisectrice
  Sample diagonal(2, 2);
  Point point(2);
  diagonal(0, 0) = data(0, 0);
  diagonal(0, 1) = data(0, 0);
  diagonal(1, 0) = data(size - 1, 0);
  diagonal(1, 1) = data(size - 1, 0);
  Curve bisectrice(diagonal, "Test line");
  bisectrice.setColor("red");
  bisectrice.setLineStyle("dashed");
  graphQQplot.add(bisectrice);
```

**Comment by jschueller** [comment]
this is a question for @sofianehaddad

**Comment by sofianehaddad** [comment]
Hi @josephmure I do not remember why it was done like that ☹️ I guess the purpose was to introduce some robustness and mimic `R` results but I am not sure
I suggest to rely on `VisualTest::DrawQQPlot`. What do you think ?

**Comment by josephmure** [comment]
Yes, I agree that `LinearModelAnalysis::drawQQPlot` should rely on `VisualTest::drawQQPlot`: there is no need to implement `DrawQQPlot` more than once.

**Comment by sofianehaddad** [comment]
Fix issue in #1584. Hereafter the current output (with the same script)

<img width="400" alt="image" src="https://user-images.githubusercontent.com/1433220/92120246-1862f700-edf9-11ea-9e70-d6b07c3373a4.png">
